roles/ngi-rnaseq/tasks/main.yml

```diff
@@ -0,0 +1,42 @@
+---
+
+- name: Download & install NextFlow 
+  shell: "curl -fsSL get.nextflow.io | bash"
+  args:
+    chmod: "{{ root_path }}/sw"
+  tags: ngi-rnaseq
+
+- name: Add nextflow to $PATH
+  lineinfile: dest="{{ ngi_pipeline_conf }}/{{ bash_env_script }}"
+              line="export PATH={{ root_path }}/sw/nextflow:$PATH"
```

Comment:
This export is only correct if you put the binary in a subdir called `nextflow`, as I contemplated above.